client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/staticstate/ExecutionTracer.java

```diff
@@ -91,7 +91,7 @@ public class ExecutionTracer {
 
     private static final Object lock = new Object();
 
-
+    private static final List<String> skippedHostName = new CopyOnWriteArrayList<>();
```

Comment:
provide docs for this. eg, why are there case in which we dynamically keep track of hostnames to skip? also, being a list, should end with a plural `s`

Reply:
also, any reason why it is not handled in `reset()`?